frontend/src/cljs/imas_seamap/map/utils.cljs

```diff
@@ -279,3 +279,56 @@
         basemap-groups (concat basemap-groups ungrouped-layers-groups)
         basemap-groups (sort-by (juxt #(or (:sort_key %) "zzzzzzzzzz") :id) basemap-groups)]
     basemap-groups))
+
+;; We need a better place for these - our goal is to genericize the catalogues
```

Comment:
Indeed -- I might write a ticket for that in fact, if you're idle!

Reply:
Good idea – it's not like I have anything better to do 😉